src/HubConnection.php

```diff
@@ -261,15 +264,17 @@ public static function get_auth_token() {
 	public function get_core_data() {
 		global $wpdb, $wp_version;
 
-		return array(
+		$data = array(
 			'url'         => get_site_url(),
 			'php'         => phpversion(),
 			'mysql'       => $wpdb->db_version(),
 			'wp'          => $wp_version,
 			'plugin'      => BLUEHOST_PLUGIN_VERSION,
 			'hostname'    => gethostname(),
 			'cache_level' => intval( get_option( 'endurance_cache_level', 2 ) ),
-			'cloudflare'  => get_option( 'endurance_cloudflare_enabled', false ),
+			'cloudflare'  => get_option( 'endurance_cloudflare_enabled', false )
 		);
+
+		return $data;
```

Comment:
We can probably skip these changes in `get_core_data()`. 
Putting trailing commas arrays is a good practice for ensuring we don't forget one and cause a fatal error if something gets added after it.
And we don't need to store this as a variable since we aren't using it. Just returning it directly gives slight reduction in memory footprint. It's definitely a micro-optimization, but not making an unnecessary changes also keeps the file/line history and thus `git blame` a little cleaner.

Reply:
Yea, good call @earnjam. This went through a couple variations and in the end didn't actually change in effect, so I can revert `get_core_data` back to how it was for simplicity.